rollup-bridge-contracts/contracts/bridge/l1/L1BridgeMessenger.sol

```diff
@@ -316,12 +316,16 @@ contract L1BridgeMessenger is
     // Check if the deposit message exists
     DepositMessage storage depositMessage = depositMessages[messageHash];
     if (depositMessage.expiryTime == 0) {
-      revert DepositMessageDoesNotExist(messageHash);
+      revert DepositMessageDoesNotExist();
     }
 
     // Check if the deposit message is already canceled
     if (depositMessage.isCancelled) {
```

Comment:
It seems like we don't need `isCancelled` anymore since it is already indicated in `isClaimed`.  Maybe just to use a generic variable name, we can use the name `isUnlocked` instead of `isClaimed` in both `cancelDeposit` and `claimFailedDeposit` functions.

Reply:
redundant isCancelled will be removed. we can depend on `isClaimed` for predicates in both `cancelDeposit` and `claimFailedDeposit` function calls